cypress/e2e/cart.cy.js

```diff
@@ -0,0 +1,66 @@
+describe('Error with no data',()=>{
```

Comment:
Refactor code to have just 1 describe with 4 it() (also use beforeEach in that case)